tracing-appender/src/lib.rs

```diff
@@ -161,6 +161,13 @@ mod worker;
 
 pub(crate) mod sync;
 
+mod writer;
+
+#[cfg(feature = "compression_gzip")]
+mod compression;
+
+pub mod builder;
```

Comment:
minor style nit: since the compression and builder modules are only used with the rolling file appender, I think they probably should be submodules of the `rolling` module, rather than modules in the crate's root...

also, we probably don't need to make the `builder` module public, since it only contains one type (`RollingFileAppenderBuilder`). instead, that type should probably just be re-exported publicly in the `rolling` module.

Reply:
I will create a new module `builder`.